$OpenBSD: patch-src_bootstrap_native_rs,v 1.3 2017/03/27 12:35:31 danj Exp $
use default settings from devel/llvm to local embedded version
--- src/bootstrap/native.rs.orig	Fri Mar 10 23:55:21 2017
+++ src/bootstrap/native.rs	Sun Mar 12 09:08:11 2017
@@ -90,16 +90,21 @@ pub fn llvm(build: &Build, target: &str) {
        .define("LLVM_INCLUDE_EXAMPLES", "OFF")
        .define("LLVM_INCLUDE_TESTS", "OFF")
        .define("LLVM_INCLUDE_DOCS", "OFF")
-       .define("LLVM_ENABLE_ZLIB", "OFF")
-       .define("WITH_POLLY", "OFF")
+       .define("LLVM_ENABLE_FFI", "OFF")
+       .define("LLVM_REQUIRES_RTTI", "ON")
+       .define("GO_EXECUTABLE", "GO_EXECUTABLE-NOTFOUND")
        .define("LLVM_ENABLE_TERMINFO", "OFF")
-       .define("LLVM_ENABLE_LIBEDIT", "OFF")
        .define("LLVM_PARALLEL_COMPILE_JOBS", build.jobs().to_string())
        .define("LLVM_TARGET_ARCH", target.split('-').next().unwrap())
        .define("LLVM_DEFAULT_TARGET_TRIPLE", target);
 
     if target.starts_with("i686") {
         cfg.define("LLVM_BUILD_32_BITS", "ON");
+    }
+
+    // propagate verbose flag to llvm build
+    if build.flags.verbose > 0 {
+        cfg.define("CMAKE_VERBOSE_MAKEFILE", "ON");
     }
 
     // http://llvm.org/docs/HowToCrossCompileLLVM.html
